### DISCUSSION TITLE
System Informer 3.1.24258 Canary

### DISCUSSION THREAD

**Discussion by leland37** [winsiderss/systeminformer#2206] [Questions / Answers] [answered]
I just spent several minutes trying to get the new canary to run on Windows 10.  It fails and closes every time even with admin access.  The same computer run canary 3.1.24244 without issue.  Windows 11 x64 runs 3.1.24258 Canary without issue.  I suspect some problem when the code was compiled.

**Comment by dmex** [comment, marked as answer]
Please create a bug report if it's still an issue: https://github.com/winsiderss/systeminformer/issues
